src/util.ts

```diff
@@ -479,6 +480,64 @@ export function thisExtensionPackage(): PackageJSON {
     };
 }
 
+export async function GetExtensionLocalizedPackageJson(): Promise<{[key: string]: any}> {
```

Comment:
Cool again! I honestly don't know if I would have thought of this. 

I assume that we need this in order to make sure the strings we output are actually localized? I do wonder if there is any way that VSCode allows for you to grab this without having to do this way, but either way, if it works, this is cool!

Reply:
From my research it doesnt look like there's a way. the loc file is really meant to localize the package.json itself